playbooks/power-linux-configure.yml

```diff
@@ -14,31 +14,39 @@
 # limitations under the License.
 # ------------------------------------------------------------------------
 ---
-- hosts: all
+- name: Executing roles for PowerVS swap and fs creation, preparing RHEL or SLES for SAP
+  hosts: all
   roles:
-  - { role: ../roles/powervs_swap_creation, when: swap_disk_wwn is defined }
-  - { role: ../roles/powervs_fs_creation, when: disks_configuration is defined }
-  - { role: ../roles/powervs_prepare_rhel_sap, when: sap_solution is defined and ansible_distribution is match("RedHat*") }
-  - { role: ../roles/powervs_prepare_sles_sap, when: sap_solution is defined and ansible_distribution is match("SLES*") }
+    - role: powervs_swap_creation
+      when: swap_disk_wwn is defined
```

Comment:
I think this roles path must be reverted. As roles will not be installed on os and will not be found. But run a test and check @AnaBil-k

Reply:
Change it to ibm.power_linux_sap.powervs_swap_creation